edition = "2023";
package external.core.create_action.v1;

message CreateActionRequest {
  string action_name = 1; // e.g.: "Do Stuff"
  string domain_path = 2; // e.g.: "mhs/example" (does not have to exist)
}

message CreateActionResponse {
  repeated string errors = 1;
}